main.go

```diff
@@ -68,9 +72,15 @@ func main() {
 	for scanner.Scan() {
 		line := scanner.Text()
 
-		if line[0] == 35 {
+		if line[0] == '#' {
+			if line == "## explicit" {
+				continue
+			}
 			s := strings.Split(line, " ")
 
+			if len(s) < 3 {
+				panic(fmt.Sprintf("could not parse modules.txt line %q: expected at least three words", line))
```

Comment:
We don't guarantee not to add a line of this form in the future. Better to skip the line instead.